terraform/modules/galileo-eks/examples/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "environment" {
+  description = "The deployment environment (e.g., staging, production)"
+  type        = string
+
+  validation {
+    condition     = contains(["staging"], var.environment)
+    error_message = "The environment must be either 'staging' or 'production'."
+  }
+}
```

Comment:
Just confirming we don't want a default here?

I expect most users of these blueprints to deploy it to their prod envs, so we should maybe not even allow setting this.

Reply:
With no default specified, Terraform/OpenTofu will ask what the value should be every time the script is run.  To me, this seemed like the best way to introduce the flexibility of deploying to multiple environments safely.